lib/Takeaway.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'Text'
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)